Docs/migration-guide-2.x-3.x.md

```diff
@@ -0,0 +1,128 @@
+## 2.x to 3.x Migration Guide
```

Comment:
Case mismatch - in some places it's `3.x`, in other places it is `3.X`.

Reply:
In some cases 3.x made sense ie in the migration guide. In the new features, it's 3.0

Reply:
I think @kbagchiGWC is referring to `3.x` (lowercase x) vs `3.X` (uppercase). The casing is different in some places in the doc.

Reply:
Ohh ok got it. Ill update to lower case